<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClientPaymentWidget</class>
 <widget class="QDialog" name="ClientPaymentWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>629</width>
    <height>589</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="mainPaymentLayout">
     <item>
      <layout class="QHBoxLayout" name="headerLayout">
       <item>
        <widget class="QLabel" name="companyLogoLabel">
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap>../images/Image3.png</pixmap>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Policy::Minimum</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="welcomeClientLabel">
         <property name="text">
          <string>Welcome Client</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="summaryGroupBox">
       <property name="title">
        <string>Invoice Summary Area</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <layout class="QGridLayout" name="summaryLayout">
          <item row="0" column="1">
           <widget class="QLabel" name="invoiceNumberDisplay">
            <property name="text">
             <string>InvoiceNumberDisplay</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="paymentAmountDueLabel">
            <property name="text">
             <string>Amount Due:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="paymentIssueDateLabel">
            <property name="text">
             <string>Issue Date:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLabel" name="dueDateDisplay">
            <property name="text">
             <string>DueDateDisplay</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="amountDueDisplay">
            <property name="text">
             <string>AmountDueDisplay</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLabel" name="issueDateDisplay">
            <property name="text">
             <string>IssueDateDisplay</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="paymentDueDateLabel">
            <property name="text">
             <string>Due Date:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="paymentInvoiceNumberLabel">
            <property name="text">
             <string>Invoice Number:</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QPushButton" name="downloadPdfButton">
            <property name="text">
             <string>Download PDF</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="paymentMethodGroupBox">
       <property name="title">
        <string>Payment Method Section</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QVBoxLayout" name="paymentMethodLayout">
          <item>
           <widget class="QLabel" name="chooseMethodLabel">
            <property name="text">
             <string>Choose Method:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="payByPayPalRadio">
            <property name="text">
             <string>PayPal</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="payByCardRadio">
            <property name="text">
             <string>Credit Card</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="payByBankTransferRadio">
            <property name="text">
             <string>Bank Transfer</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QStackedWidget" name="paymentDetailsStack">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="payPalPage">
           <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="0">
             <widget class="QLabel" name="payPalInfoLabel">
              <property name="text">
               <string>PayPal Info</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QPushButton" name="payPalPayButton">
              <property name="text">
               <string>Pay WithPalPay</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="cardPaymentPage">
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <layout class="QFormLayout" name="formLayout">
              <item row="0" column="0">
               <widget class="QLabel" name="cardNumberLabel">
                <property name="text">
                 <string>Card Number:</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLineEdit" name="cardNumberInput"/>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="expiryDateLabel">
                <property name="text">
                 <string>Expiry Date:</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QLineEdit" name="expiryDateInput"/>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="cVVLabel">
                <property name="text">
                 <string>CVV:</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QLineEdit" name="cVVInput"/>
              </item>
              <item row="3" column="0">
               <widget class="QLabel" name="cardholderNameLabel">
                <property name="text">
                 <string>Cardholder Name:</string>
                </property>
               </widget>
              </item>
              <item row="3" column="1">
               <widget class="QLineEdit" name="cardholderNameInput"/>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="bankTransferPage">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <layout class="QVBoxLayout" name="bankInfoLayout">
              <item>
               <widget class="QTextEdit" name="bankInfoDisplay"/>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="actionLayout">
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QProgressBar" name="paymentProgressBar">
         <property name="value">
          <number>24</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="errorMessageLabel">
         <property name="text">
          <string>........</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="proceedToPaymentButton">
         <property name="text">
          <string>Proceed To Payment</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
